packages/react-instantsearch/rollup.config.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import babel from 'rollup-plugin-babel';
+import globals from 'rollup-plugin-node-globals';
+import replace from 'rollup-plugin-replace';
+import uglify from 'rollup-plugin-uglify';
+import filesize from 'rollup-plugin-filesize';
+/* eslint-enable import/no-extraneous-dependencies */
```

Comment:
I think those comments can simply be removed, since you use them later 🤔

Reply:
> Why not add this to our global eslint rules? https://github.com/algolia/eslint-config-algolia/

Actually the rule is pretty useful most of the time. We will see if the resolution problem is still present with the workspaces.

> I think those comments can simply be removed, since you use them later 🤔

I don't understand your remark. If I remove the comment ESLint will trigger warning about external dependencies import. It's because modules are installed in the root folder instead of this one.

Reply:
misunderstood which rule was disabled; lgtm